app/lib/profile/post_history.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+class PostHistory extends StatefulWidget {
+  final List<Post> posts;
+  final bool isPersonalProfile;
+
+  PostHistory({ this.posts, this.isPersonalProfile });
+
+  @override
+  _PostHistoryState createState() => _PostHistoryState();
+}
+
+class _PostHistoryState extends State<PostHistory> {
+  @override
+  Widget build(BuildContext context) {
+    bool hasPosts = widget.posts != null && widget.posts.length > 0;
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        hasPosts ? Text('${widget.posts.length} julkaisua',
+            textAlign: TextAlign.left,
+            style: Theme.of(context).textTheme.title,
+        ) : Container(),
+
+        SizedBox(height: 28,),
+
+        widget.posts == null || widget.posts.length == 0 ? Center(
```

Comment:
voisko näitä vähän siistiä johonki erillisiin muuttujiin tms, että vähentäis tämmöstä inline-koodia, tyyliin javascriptissä

```
const isPosts = widget.posts == null || widget.posts.length == 0;
const posts = isPosts ? ... : ...;

return Stuff(Puff(posts));
```